api/tests/integration/UserTest.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class UserTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->jwtAuth = $this->app->make('Tymon\JWTAuth\JWTAuth');
+    }
+
+    protected function assertException($message, $statusCode, $exception)
+    {
+        $body = json_decode($this->response->getContent());
+        $this->assertResponseStatus($statusCode);
+        $this->assertContains($message, $body->message);
+        $this->assertContains($exception, $body->debug->exception);
+    }
+
+    protected function createUser($type = 'admin')
+    {
+        $user = factory(App\Models\User::class)->make();
```

Comment:
This could be simplified here, and in all the tests to `$user = factory(App\Models\User::class)->create(['user_type' => $type]);`